modules/0_git/test/main.cpp

```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```

Comment:
Please revert this change (next just start every new task from master branch):
```
git checkout master
git checkout -b new_task
```

Reply:
please not forget to fix it as well

Reply:
just a moment